kpatch-build/Makefile

```diff
@@ -22,7 +22,7 @@ GCC_PLUGINS_DIR := $(shell gcc -print-file-name=plugin)
 PLUGIN_CFLAGS := $(filter-out -Wconversion, $(CFLAGS))
 PLUGIN_CFLAGS += -shared -I$(GCC_PLUGINS_DIR)/include \
 		   -Igcc-plugins -fPIC -fno-rtti -O2 -Wall
-else
+else ifneq ($(ARCH),s390x)
 $(error Unsupported architecture ${ARCH}, check https://github.com/dynup/kpatch/#supported-architectures)
```

Comment:
Depending on how confident we feel after this PR, we could update the referenced README.md `#supported-architectures` section to include s390x.  Or would you prefer we merge and test internally before advertising support.

Reply:
After this is merged, we can update s390x as supported in README.md.